DNN Platform/Library/Security/Permissions/Controls/PermissionsGrid.cs

```diff
@@ -1113,6 +1113,11 @@ private void FillSelectRoleComboBox(int selectedRoleGroupId)
                 groupRoles.Add(new RoleInfo { RoleID = this.AllUsersRoleId, RoleName = Globals.glbRoleAllUsersName });
             }
 
+            foreach (var role in groupRoles)
+            {
+                role.RoleName = Localization.LocalizeRole(role.RoleName);
+            }
+

```

Comment:
Suggested to move this into next `foreach`

```suggestion
```

Reply:
See above